### PR TITLE
used planetarium tx library and modified updatedAddress

### PR DESCRIPTION
**Pull request by ahramyun7**
- In order to use planetarium tx library, node version > 19.xx is required. => updated to node 20.x
- Updated Dockerfile
- Updated package.json and dependencies
- Added updatedAddress set

Test DONE:

* Bridge Tx: https://planetariumhq.slack.com/archives/C03S6FD3N92/p1689581789441079
Screenshot 2023-07-17 at 5 16 51 PM

* GraphQL updatedAddress:
Screenshot 2023-07-17 at 5 16 47 PM

### CODE REVIEW

**Review by longfin — APPROVED** [review]

File: bridge/src/headless-graphql-client.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class HeadlessGraphQLClient implements IHeadlessGraphQLClient {
         return this._apiEndpoint;
     }
 
-    async getBlockIndex(blockHash: BlockHash): Promise<number> {
+    async getBlockIndex(blockHash: BlockHash): Promise<number> {
```
Comment (moreal) [inline bridge/src/headless-graphql-client.ts:34]: This change seems not intended.

```suggestion
    async getBlockIndex(blockHash: BlockHash): Promise<number> {
```

**Review by moreal — APPROVED** [review]
🙇🏻‍♂️

File: bridge/src/headless-graphql-client.ts (hunk 1 above)
Comment (ahramyun7) [reply bridge/src/headless-graphql-client.ts]: udpated.

File: .github/workflows/build-bridge.yaml [hunk 2]
```diff
@@ -13,12 +13,15 @@ jobs:
...
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 20
       - uses: actions/cache@v2
         with:
           path: ~/.npm
           key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
           restore-keys: |
-            ${{ runner.os }}-node-
+            ${{ runner.os }}-node-
```
Comment (moreal) [inline .github/workflows/build-bridge.yaml:24]: This change seems not intended.

```suggestion
            ${{ runner.os }}-node-
```

**Review by moreal — APPROVED** [review]

**Review by moreal — APPROVED** [review]

**Review by shkangr — APPROVED** [review]

File: .github/workflows/build-bridge.yaml (hunk 2 above)
Comment (ahramyun7) [reply .github/workflows/build-bridge.yaml]: it was not intended. removed.

**Pull request merged**